### PR TITLE
chore: modify readme

### PR DESCRIPTION
**Pull request by gxthrj**

### CODE REVIEW

File: doc/design/design.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
+# ingress-controller
```
Comment (membphis) [inline doc/design/design.md:1]: file path: `doc/design.md` is fine

File: README.md [hunk 2]
```diff
@@ -1,55 +1,159 @@
...
+kubectl apply -f samples/deploy/deployment/ingress-controller.yaml
+```
+
+### Helm
+
+// todo
+
+## Document
+
+* [SDK doc](./doc/dev/develop.md)
```
Comment (membphis) [inline README.md:139]: file path: `doc/develop.md` is fine

**Review by membphis — COMMENTED** [review]
file path: `doc/images/xxxx.png` is fine

File: doc/design/design.md (hunk 1 above)
Comment (gxthrj) [reply doc/design/design.md]: fixed, now it is `docs/design.md`

File: README.md (hunk 2 above)
Comment (gxthrj) [reply README.md]: fixed, now it is `docs/develop.md`

**Review by membphis — APPROVED** [review]
LGFM

**Pull request merged**